src/utils/IdealGraphVisualizer/Filter/src/main/java/com/sun/hotspot/igv/filter/CombineFilter.java

```diff
@@ -75,8 +70,8 @@ public void apply(Diagram diagram) {
                         }
 
```

Comment:
Makes sense, but I did not change it because the surrounding code is essentially dead (no current filter has a "reversed" `CombineRule`) and I would not be able to test it. Since this code has not been executed for years, it is likely to be broken anyway.

Reply:
okey